packages/@uppy/transloadit/src/Client.js

```diff
@@ -96,5 +103,41 @@ module.exports = class Client {
   getAssemblyStatus (url) {
     return fetch(url)
       .then((response) => response.json())
+      .catch((err) => this._reportError(err, { url }))
+  }
+
+  submitError (err, { endpoint, instance, assembly }) {
+    return fetch('https://status.transloadit.com/client_error', {
+      method: 'post',
+      body: JSON.stringify({
+        endpoint,
+        instance,
+        assembly_id: assembly,
+        ip: null, // can't this just be done by statuspage?
+        agent: typeof navigator !== 'undefined' ? navigator.userAgent : '',
```

Comment:
can we send the Transloadit-Client along with this?

Reply:
i'm not sure how the backend for this works, could we just add a new property like this?
```js
client: 'string header'
```

Reply:
we sure could!

Reply:
we'll add some handling to the statuspage later on :ok_hand:

Reply:
we don't actually have the version information available yet at runtime (https://github.com/transloadit/uppy/issues/1118), I made a quick implementation in [a branch](https://github.com/transloadit/uppy/tree/feature/transloadit-client) but to auto update it will require some more work that we probably can't fit in right now.

Reply:
ok, it's cool to add this later, please leave a note in the 1.1 milestone in the CHANGELOG :ok_hand: